rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,6 +16,8 @@ toc::[]
 [id="rosa-q1-2024_{context}"]
 === Q1 2024
 
+* **Log linking is enabled by default** - Beginning with {product-title} 4.15, log linking is enabled by default. Log linking gives you access to the container logs for your pods.
```

Comment:
No _For more information_ link ike the others? (not that I know where to link)

Reply:
Not that I can think of. We just had it as an activated feature. Not much more we can do there.